Automattic/pocket-casts-ios
# Rename image

Rename an image to avoid duplication

## To test

- CI must be 🟢 
- Enable the FF `winback`
- Use a Plus account and open the Cancel Subscription in Account
- Double check the chevron icons of the rows still display

<img width="390" alt="Screenshot 2024-11-25 at 10 25 57" src="https://github.com/user-attachments/assets/2216c87a-015b-42ed-90b9-7fd29d872f1e">


## Checklist
